frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -51,6 +59,7 @@ const ImageStreamTagDropdown: React.FC = () => {
           setFieldValue('isi.tag', selectedTag);
           setFieldValue('isi.ports', ports);
           setFieldValue('image.ports', ports);
+          setFieldValue('labels', imgStreamLabels);
```

Comment:
I dont think this is the correct way to add labels to the app resources created uisng internal option since if you add labels like this they are considered as labels added by the user and can be removed from within the form which should not be the case here since these labels are added by us.

Reply:
I agree, these labels are meant for user to override or provide new values. so changed the logic to fetch the imagestream name from the existin labels